### ISSUE TITLE
Focus outline on slider items is cut out at the top and bottom

### ISSUE THREAD

**Issue by ludoboludo** [Shopify/dawn#1157]
**Describe the current behavior**

Focus outlines are cut out a bit at the top and bottom:
![](https://screenshot.click/22-01-p00oi-zthd1.png)

**Describe the expected behavior**

We should be able to see the focus ring properly.

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 2.5.0
